### ISSUE TITLE
Syntacore custom modification problems

### ISSUE THREAD

**Issue by GARIIKK** [syntacore/scr1#42]
Hello, 
I have taken bare SCR1 core, then add my own RAM and ROM and create wishbone wrapper in order to use wishbone as a communication BUS. I have also generated .memh files with my soft for every extension I needed and load it to ROM. Also instantiate some necessary defines in my tb.  But I have some problem with instruction receiving from ROM. PC stucks in 0x1C0 value immediately after first instruction request. I attached some screenshots of my defines, .memh file and waveforms with stucked  PC
![image](https://user-images.githubusercontent.com/25632716/206496352-d52dcd7f-5857-4d8b-a070-d196ab3def5d.png)
As you can see on screenshot below, boot addr started from 0x80, so I changed SCR1_ARCH_RST_VECTOR = 0x80.
![image](https://user-images.githubusercontent.com/25632716/206500455-8f2fef7f-12f4-4111-ba2e-afd97a931c75.png)
To summarize, I can read first instruction from ROM but PC stucks even before cpu2mem request.
![image](https://user-images.githubusercontent.com/25632716/206504150-de86d1a6-7071-498b-8121-395f01797ebe.png)
Could you please advise me something, probably I forgot to instantiate some defines or so?

**Comment by achuykov-sc** [comment]
@GARIIKK, 

Could you please check the linker script for your application (CoreMark)? 
The default entry point is 0x200. So, if you had changed the reset vector address, you should change linker script according to your changes. 

Best regards,
Alexander

**Comment by GARIIKK** [comment]
Hello @achuykov-sc ,
Sorry for delayed answer. I have found the problem. It is closely connected with my SyntaCore2Wishbone bridge. I reread your documentation on AHB and AXI bus. But I still cant understand how I should do this communication properly. I did some changes with delaying(pipelining) of some request/response signals to/from Wishbone BUS. After this modification Core starts execute the instruction, writes and reads from Data memory. But still does it incorrectly(compare to another RISC-V core which 100% works correctly with the same soft, linker script, etc.). Could you please describe this SyntaCore2BUS communication in more details?
Regards,
Ilya

**Comment by achuykov-sc** [comment]
Hi Ilya,

Could you please more details? You had mentioned: "But still does it incorrectly". What exactly is incorrect? 
Please attach waves and logs.

Best regards,
Alexander
